### ISSUE TITLE
[NEWS] Jeśli nie jesteś właścicielem oferty, nie zwracamy wartości kodu EAN w metodach doShowItemInfoExt i doGetItemsInfo

### ISSUE THREAD

**Issue by TomekKaczmarek** [allegro/allegro-api#879]
Zgodnie z naszą zapowiedzią - od dziś, jeśli nie jesteś zalogowany, jako sprzedawca, który wystawił ofertę, nie zwrócimy wartości kodu EAN w metodach [doShowItemInfoExt](https://allegro.pl/webapi/documentation.php/show/id,342) i [doGetItemsInfo](https://allegro.pl/webapi/documentation.php/show/id,52). W odpowiedzi wartość pola EAN zawsze będzie pusta.

**Comment by rtnet-pl** [comment]
@TomekKaczmarek Niepotrzebnie powstała burza i chaos informacyjny wokół zmian związanch z ean, w poprzednim komunikacie informowaliście że usuwacie ean z doGetItemsInfo i doShowItemInfoExt. Tymczasem ean-y dalej tam są tam gdzie były, usunęliście tylko widoczność eanów w obcych ofertach. 
Rozumiem, że tak zostaje?

**Comment by TomekKaczmarek** [comment]
@rtnet-pl zgadza się. Biję się w pierś, nastąpiła niespójność w komunikacji - sprawę wyjaśniamy, wyciągniemy wnioski, postaramy się w przyszłości unikać. Przepraszam za zamieszanie.

W każdym razie, jeśli jesteś zalogowany jako sprzedawca, który wystawił ofertę - w odpowiedzi w tych dwóch metodach otrzymasz właściwą wartość kodu EAN. Jeśli pobierasz dane o ofertach, których nie wystawiłeś, wtedy wartość pola EAN będzie pusta.

**Comment by FromAnyHole** [comment]
@TomekKaczmarek
Czyli warto było założyć wątek i napisać o próbie przewalenia przez Was tej sprawy:
https://github.com/allegro/allegro-api/issues/860
p.s.
Do wszystkich czytających - reagujcie tu podobnie i wspierajcie nawet te sprawy, które Was na dany moment nie dotyczą, bo Allegro ma z nas Full Hajcu i robi zwykle tylko sobie dobrze ! Nie dziękujcie ;-)
